Documentation/Scripts/exampleHeader.js

```diff
@@ -63,15 +63,15 @@ function normalize(lang) {
     return MAP[lower] || lower;
 }
 
-function highlight(lang, code) {
-  if(!lang) {
+function highlight(language, code) {
+  if(!language) {
     return code;
   }
-  // Normalize lang
-  lang = normalize(lang);
+  // Normalize language
+  language = normalize(language);
 
   try {
-    return hljs.highlight(lang, code).value;
+    return hljs.highlight(code, {language}).value;
```

Comment:
IDK, we should ask @dothebart

Reply:
If I had one wish it's throwing out all our jslint nonsense and just using an external eslint+prettier via npm. We could probably also do this automatically for PRs.

Reply:
Yes, I bet our jslint rules aren't the best. Personally I do not care which tool we're going to use. Also adding jslint support for documentation related JS files is out of scope here. But please make sure that this will not be forgotten. 

@Simran-B fyi.

Reply:
@Simran-B do you have any insight what this documentation file does and why we bundle `highlight.js`? If this code is unused we could remove the dependency.

Reply:
To me it looks like the file is used in `generateExamples.py` so apparently it's part of the docs build process? Seems a bit extreme to bundle and distribute JS dependencies that are only ever used once during build and then never again.